xarray/tests/test_variable.py

```diff
@@ -1077,6 +1077,17 @@ def test_repr(self):
         ).strip()
         assert expected == repr(v)
 
+    def test_repr_html(self):
+        v = Variable(["time", "x"], [[1, 2, 3], [4, 5, 6]], {"foo": "bar"})
+        assert hasattr(v, "_repr_html_")
+        with set_options(display_style="html"):
```

Comment:
There's no need to set the display option here if you're calling `_repr_html_` direclty.

Reply:
Are you sure? There's an OPTIONS switch in AbstractArray._repr_html_() which
might break the test if it's not set to "html".

Reply:
I think what @shoyer meant was that you should directly use `formatting_html.array_repr` instead of `_html_repr_` so there's no need for the options switch (you can keep the `hasattr` check, though).

Reply:
OK, good catch. This is fine then.